specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1146,34 +1147,34 @@ The JDBC 4.2 specification is available at
 _https://jcp.org/en/jsr/detail?id=221_ .
 
 [[a2553]]
-===== Jakarta™ API for XML Web Services (JAX-WS) Requirements
+===== Jakarta™ XML Web Services Requirements
```

Comment:
@bshannon also commented on this...  JAX-WS is part of Java SE, not Jakarta EE.  Not as a spec.  The original title for this section was "Java API for XML Web Services (JAX-WS)".  This is just like JDBC and Java IDL...  Don't change anything in this sub-section on JAX-WS.